DancingGoat/Helpers/Extensions/HtmlHelperExtensions.cs

```diff
@@ -46,17 +48,19 @@ public static MvcHtmlString AssetImage(this HtmlHelper htmlHelper, Asset asset,
         /// <summary>
         /// Displays a <see cref="DateTime"/> in a formatted manner.
         /// </summary>
-        /// <param name="htmlHelper">HTML helper.</param>
+        /// <param name="htmlHelper">HTML helper</param>
+        /// <param name="expression">The expression of the model property</param>
         /// <param name="dateTime">The <see cref="DateTime"/> to format</param>
         /// <param name="format">The formatting character</param>
-        public static MvcHtmlString DateTimeFormatted(this HtmlHelper htmlHelper, DateTime? dateTime, string format)
+        /// <remarks>The TValue generic parameter is chosen instead of DateTime just to save views from falling to exceptions. With TValue, the views will get rendered, only this helper method will return an empty <see cref="MvcHtmlString"/>.</remarks>
+        public static MvcHtmlString DateTimeFormattedFor<TModel, TValue>(this HtmlHelper<TModel> htmlHelper, Expression<Func<TModel, TValue>> expression, string format)
```

Comment:
I was also tempted to remove it. But the TValue parameter is used to 'bind' the System.Linq.Expression of the stack DisplayFor method with the expression that my method. I could change it to `public static MvcHtmlString DateTimeFormattedFor<TModel, DateTime>(this HtmlHelper<TModel> htmlHelper, Expression<Func<TModel, DateTime>> expression, string format)`, but, anybody could still pass whatever type to the method. And this would make the whole view crash without rendering anything.